cmake/unittest.cmake

```diff
@@ -62,6 +62,10 @@ function(AddTest)
     target_link_libraries(${_UT_TARGET} ${PROTOBUF_LIBRARIES})
   endif()
 
+  if(ONNX_DUMMY_BACKEND)
+	  target_compile_definitions(${_UT_TARGET} PUBLIC "ONNX_DUMMY_BACKEND=true")
```

Comment:
Don't you need "-DONNX_DUMMY_BACKEND"? (but i'm not 100% certain)

Reply:
Actually whether -D or not it doesnt works...